packages/controller/lib/setup/setupUpgrade.js

```diff
@@ -149,13 +125,13 @@ function Upgrade(options) {
 
             debug(`upgrade order is "${sortedAdapters.join(', ')}"`);
 
-            await this._upgradeAdapterHelper(repo, sortedAdapters, 0, forceDowngrade, autoConfirm);
```

Comment:
this method was no longer existing

Reply:
Yes. It was  replaced by "for" cycle and no more required